### PR TITLE
Trim whitespace from email checks

### PR DESCRIPTION
**Pull request by pdehaan**
`trim()` this, `trim()` that...

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ function loadLists() {
 		}
 
 		files.forEach(function(file) {
-			readFile(pathToLists + file);
+			readFile(path.join(pathToLists, file));
```
Comment (pdehaan) [inline index.js:18]: Oops, missed this in previous PR.

File: index.js [hunk 2]
```diff
@@ -34,7 +34,7 @@ function readFile(file) {
 				case '\n':
 					break;
 				default:
-					lists.push(email);
+					lists.push(email.trim());
```
Comment (pdehaan) [inline index.js:37]: `trim()` the email address from the static file before adding it to the list.

File: index.js [hunk 3]
```diff
@@ -43,6 +43,6 @@ function readFile(file) {
 
 module.exports = {
 	isEmailClean: function(email) {
-		return (lists.indexOf(email) === -1);
+		return (lists.indexOf(email.trim()) === -1);
```
Comment (pdehaan) [inline index.js:46]: `trim()` the user supplied email address when comparing it to the known-good list.

**Comment by adityabansod** [comment]
you're having quite some fun with this, aren't you? ;-)

**Pull request merged**